compose

```diff
@@ -20,6 +21,9 @@ EOF
 fi
 
 echo "INFO: Using compose profile ${COMPOSE_PROFILE}"
+echo "INFO: Installing from source ${DEV_SOURCE_PATH}"
+echo "INFO: Image suffix ${DEV_IMAGE_SUFFIX}"
+echo "INFO: Volume suffix ${DEV_VOLUME_SUFFIX}"
```

Comment:
I'd be nice to print debug log to `stderr`.

Reply:
@cutwater what is the advantage of routing it to stderr?

Reply:
Could `DEV_IMAGE_SUFFIX` and `DEV_VOLUME_SUFFIX` be given default values?

Reply:
@ironfroggy it already defaults to Blank Value so it keeps compatibility with current environments that has no suffix.